rotkehlchen/tests/unit/decoders/test_curve.py

```diff
@@ -2404,3 +2404,63 @@ def test_gauge_deposit_and_stake_multiple(database, gnosis_inquirer, gnosis_acco
             address=ZERO_ADDRESS,
         ),
     ]
+
+
+@pytest.mark.vcr()
+@pytest.mark.parametrize('load_global_caches', [[CPT_CURVE]])
+@pytest.mark.parametrize('gnosis_accounts', [['0xc37b40ABdB939635068d3c5f13E7faF686F03B65']])
+def test_liquidity_withdrawal(database, gnosis_inquirer, gnosis_accounts, load_global_caches):
+    """Test that a withdrawal in the case of pools that have underlying pools
+    is correctly decoded"""
+    tx_hash = deserialize_evm_tx_hash('0x7645bfeb43a1ccacd3f8eb29c496823bd73586498ff7b79be5a3a48145f1c4b4')  # noqa: E501
+    timestamp, pool_address, gas_fees, removed_amount = TimestampMS(1719046465000), string_to_evm_address('0x0CA1C1eC4EBf3CC67a9f545fF90a3795b318cA4a'), '0.000813878', '1656.600276747801451581'  # noqa: E501
+    returned_amount = '850'
```

Comment:
Please be combining them.

```suggestion
    timestamp, pool_address, gas_fees, removed_amount, returned_amount = TimestampMS(1719046465000), string_to_evm_address('0x0CA1C1eC4EBf3CC67a9f545fF90a3795b318cA4a'), '0.000813878', '1656.600276747801451581', '850'  # noqa: E501
```